webdriver-spec.html

```diff
@@ -1516,7 +1524,119 @@ <h3 id="page-load-strategies">The Page Load Strategy</h3>
        <li>Return the result of running the <a>remote end steps</a> for the <a>Get Window Handles</a> command.
      </ol>
    </section>
-  </section>
+
+   <section>
+     <h3>Switch to Frame</h3>
+     <table class="simple jsoncommand">
+       <tr>
+         <th>HTTP Method</th>
+         <th>Path Template</th>
+         <th>Notes</th>
+       </tr>
+       <tr>
+         <td>POST</td>
+         <td id='post-frame'>/session/{sessionId}/frame</td>
+         <td></td>
+       </tr>
+     </table>
+     <p>The <a>Switch to Frame</a> command is used to select a child
+     browsing context of the <a>current browsing context</a> to use as
+     the <a>current browsing context</a> for subsequent commands.</p>
+     <p>The <a>remote end steps</a> for the <dfn>Switch to Frame</dfn>
+     command are:</p>
+     <ol>
+
+       <li><p>If the <a>current browsing context</a> is <a>no longer open</a>,
+           return an error with code <a>no such window</a>.</li>
+
+       <li><p>Let <var>id</var> be the result of <a>getting a property</a> named "id"
+           from the <var>parameters</var> argument.</li>
+
+       <li>
+         <p>If <var>id</var> is null:
+           <ol><li><p>Set the <a>current browsing context</a> to
+                 the <a>current top-level browsing context</a>.</li>
+           </ol>
+         <p>Otherwise if <var>id</var> is a <code>Number</code> object:</p>
+         <ol>
+           <li><p>If <var>id</var> is less than 0 or greater than
+           2<sup>16</sup> - 1, return <a>error</a> with status <a>no
```

Comment:
No idea, I was just copying what the IDL used to say about this being a short. It doesn't seem particularly sane to me to have this restriction.